terraform/projects/app-ecs-services/config/alerts-tests/README.md

```diff
@@ -0,0 +1,108 @@
+# Writing Tests for Alerts
```

Comment:
I don't think we want these files duplicated in both `terraform/projects/app-ecs-services...` and `terraform/modules/app-ecs-services`?

Reply:
They should be deleted